prov/efa/src/efa_prov_info.c

```diff
@@ -722,3 +722,21 @@ int efa_prov_info_compare_src_addr(const char *node, uint64_t flags, const struc
 	return 0;
 }
 
+/*
+ * @brief If domain name of the EFA device is specified (using -d <domain_name>),
```

Comment:
It is not necessary to say `-d`, which is a fabtests behavior, and not related to the behavior.

Should be something like 
```
check whether hints's domain name setting matches prov_info
```